docs/mavlink-integration.md

```diff
@@ -0,0 +1,23 @@
+## MAVLink Integration
+
+Vehicles that run *ArduPilot* (E.g: **ArduSub** and *ArduRover*) communicates with a protocol called **MAVLink**,
+Some supported **Ping Viewer** sensors can communicate with these vehicles to fetch the necessary information to improve,
+the visualization experience and provide extra information about the environment and perspective.
+
+### MAVLink Source
+
+Ping Viewer by default expects to communicate with a vehicle that contains a companion computer, or any other component
+that can provide MAVLink messages via an UDP server with the IP address **192.168.2.2** and port **14660**.
+
+> Note: The UDP server IP address and port can't be configured with the actual version of Ping-Viewer.
+
+
+### Supported Devices and Visualization Settings
+
+Right now Ping360 is the only supported device that integrates with MAVLink, the visualizer updates the heading based in
+vehicle frame to improve pilot perspective about the environment.
```

Comment:
```suggestion
vehicle orientation to improve pilot perspective about the environment.
```

Reply:
this is nitpicking, I just think this is clearer. Or add the implicit "coordinate" of "coordinates frame"?